app/components/bookmarks/types/-default.js

```diff
@@ -20,6 +20,11 @@ export default class DefaultBookmark extends Component {
       event_category: 'Bookmark',
       event_action: 'Deleted Bookmark',
     });
+    this.get('metrics').trackEvent('MatomoTagManager', {
```

Comment:
Is there a reason you left the `gtag` call above this instead of deleting it like you did with the `GoogleAnalytics` metrics call elsewhere?

Reply:
gtag is GA4, GoogleAnalytics is universal analytics, which is no longer in use.  I left in instances of GA4, but replaced the GoogleAnalytics instances with Matomo.